website/pages/docs/platform/k8s/helm/examples/ha-with-consul.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: "docs"
+page_title: "Highly Available Vault Cluster with Consul"
+sidebar_current: "docs-platform-k8s-examples-ha-with-consul"
+sidebar_title: "Highly Available Vault Cluster with Consul"
+description: |-
+  Describes how to set up a highly available Vault cluster with Consul backend
+---
+
+# Highly Available Vault Cluster with Consul
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+The below `values.yaml` can be used to set up a five server Vault cluster using
+Consul as a highly available storage backend, Google Cloud KMS for Auto Unseal.
+
+```yaml
+server:
+  extraEnvironmentVars:
+    GOOGLE_REGION: global
+    GOOGLE_PROJECT: myproject
+    GOOGLE_APPLICATION_CREDENTIALS: /vault/userconfig/my-gcp-iam/myproject-creds.json
+
+  extraVolumes: []
```

Comment:
There's a bug here: need to remove the `[]`, it will cause an error.

Reply:
Removed